### PR TITLE
new relic browser script

### PR DESCRIPTION
**Pull request by craggy2593**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -9,4 +9,9 @@
 
     <meta name = "viewport" content = "width=device-width; initial-scale = 1; maximum-scale=1; user-scalable = no;" />
 
+     <script type="text/javascript">
```
Comment (behaghel) [inline _includes/head.html]: Can we add a quick comment above to say _NewRelic agent_ or something similar?

Comment (craggy2593) [inline _includes/head.html]: Added

**Comment by behaghel** [comment]
Have you tested it? When run locally, do you see the data flowing through on NR dashboards?

**Comment by craggy2593** [comment]
Yes. When we run it locally, we are seeing data coming into each of the NR Browser pages.

**Pull request merged**